lib/public_key/test/pkits_SUITE.erl

```diff
@@ -565,31 +729,510 @@ valid_key_usage() ->
+         {"4.10.5.2",  "Valid Policy Mapping Test5 EE",
+          {bad_cert,
+           {policy_requirement_not_met,
+            {{explicit_policy,0},{policy_set,[]}}}}}
+        ]).
+
+valid_policy_mapping_6() ->
+    [{doc, ""}].
+valid_policy_mapping_6(Config) when is_list(Config)->
+    run([{"4.10.6.1", "Valid Policy Mapping Test6 EE", ok},
+         {"4.10.6.2", "Valid Policy Mapping Test6 EE",
+          {bad_cert,
+           {policy_requirement_not_met,
+            {{explicit_policy,0},{policy_set,[]}}}}}
+        ]).
+
+invalid_policy_mapping_7() ->
+    [{doc, "In this test, the intermediate certificate includes a policyMappings extension "
+      "that includes a mapping in which the issuerDomainPolicy is anyPolicy. The intermediate "
+      "certificate also includes a critical policyConstraints extension with requireExplicitPolicy "
+      "set to 0. "}].
+invalid_policy_mapping_7(Config) when is_list(Config)->
+    run([{"4.10.7", "Invalid Mapping From anyPolicy Test7 EE",
+          {bad_cert,
+           {invalid_policy_mapping,
+            #'PolicyMappings_SEQOF'{
+               issuerDomainPolicy = ?anyPolicy,
+               subjectDomainPolicy = oidify(?NIST1)
+              }}}}
+        ]).
+invalid_policy_mapping_8() ->
+    [{doc, "In this test, the intermediate certificate includes a policyMappings "
+      "extension that includes a mapping in which the subjectDomainPolicy is anyPolicy. "
+      "The intermediate certificate also includes a critical policyConstraints extension with "
+      "requireExplicitPolicy set to 0. "}].
+invalid_policy_mapping_8(Config) when is_list(Config)->
+    run([{"4.10.8", "Invalid Mapping To anyPolicy Test8 EE",
+          {bad_cert,
+           {invalid_policy_mapping,
+            #'PolicyMappings_SEQOF'{
+               issuerDomainPolicy = oidify(?NIST1),
+               subjectDomainPolicy = ?anyPolicy}}}}
+        ]).
+
+valid_policy_mapping_9() ->
+    [{doc, ""}].
+valid_policy_mapping_9(Config) when is_list(Config)->
+    run([{"4.10.9", "Valid Policy Mapping Test9 EE", ok}
+        ]).
+
+invalid_policy_mapping_10() ->
+    [{doc, ""}].
+invalid_policy_mapping_10(Config) when is_list(Config)->
+    run([{"4.10.10",  "Invalid Policy Mapping Test10 EE",
+          {bad_cert,
+           {policy_requirement_not_met,
+            {{explicit_policy,0},{policy_set,[]}}}}}
+        ]).
+
+valid_policy_mapping_11() ->
+    [{doc, ""}].
+valid_policy_mapping_11(Config) when is_list(Config)->
+    run([{"4.10.11",  "Valid Policy Mapping Test11 EE", ok}
+        ]).
+
+valid_policy_mapping_12() ->
+    [{doc, ""}].
+valid_policy_mapping_12(Config) when is_list(Config)->
+    run([{"4.10.12.1", "Valid Policy Mapping Test12 EE", ok},
+         {"4.10.12.2", "Valid Policy Mapping Test12 EE", ok}
+        ]).
+
+valid_policy_mapping_13() ->
+    [{doc, ""}].
+valid_policy_mapping_13(Config) when is_list(Config)->
+    run([{"4.10.13.1", "Valid Policy Mapping Test13 EE", ok},
```

Comment:
Section 4.10.13 describes single test case. why 2 are added here?

Reply:
Because I decide I wanted to test one more version of the valid policy tree!